src/components/Game/Game.js

```diff
@@ -22,11 +22,12 @@ class GameView extends Component {
         this.state = {
             letters: [],
             lettersInSlots: [],
+            currentAnswer: [],
             isSubmitVisible: false,
             dragDisabled: true,
```

Comment:
would it be possible to set `dragDisabled` to the desired initial value here so  `this.setDragAndDrop();` is not needed in `componentDidMount`?